src/ui/components/Header/Header.tsx

```diff
@@ -169,12 +171,24 @@ export default function Header() {
   const { recording, loading } = hooks.useGetRecording(recordingId);
   const backIcon = <div className={classNames(styles.BackButton, "img", "arrowhead-right")} />;
 
-  const dashboardUrl = window.location.origin;
-
   if (loading) {
     return <div className={styles.Header}></div>;
   }
 
+  assert(recording != null);
+
+  let dashboardUrl = window.location.origin;
+  if (recording.workspace !== null) {
+    dashboardUrl = `/team/${recording.workspace?.id}`;
+
+    if (recording.isTest && recording.metadata?.test !== undefined) {
+      const runId = getTestRunId(recording.metadata.test);
+      dashboardUrl += `/runs/${runId}`;
+    }
+  } else {
+    dashboardUrl = "/team/me/recordings";
+  }
```

Comment:
I don't love how coupled this code is with the combination of (global/Test Suite library) routing.